### PR TITLE
feat: TMYG13

### PR DESCRIPTION
**Pull request by Kouteki**
Add TMYG13 article

### CODE REVIEW

File: posts/2024-11-21_myg-13/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+publication_date: 2024-10-21T00:00:00Z
```
Comment (ltzmaxwell) [inline posts/2024-11-21_myg-13/README.md:2]:
```suggestion
publication_date: 2024-11-21T00:00:00Z
```

File: posts/2024-11-21_myg-13/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+slug: monthly-dev-13
+tags: [gnoland, ecosystem, updates, gnovm]
+authors: [Kouteki]
+---
+
+# The More You Gno 13: Test5, gnoverse and more
+
+This edition of The More You Gno is packed with updates and highlights. Test5 testnet was launched; a new community space called gnoverse is now available. The mainnet launch milestone is up and the development is speeding up. It's an exciting time to be a part of the gno.land project.
```
Comment (ltzmaxwell) [inline posts/2024-11-21_myg-13/README.md:10]:
```suggestion
This edition of "The More You Gno" is packed with updates and highlights. Test5 testnet was launched; a new community space called gnoverse is now available. The mainnet launch milestone is up and the development is speeding up. It's an exciting time to be a part of the gno.land project.
```

File: posts/2024-11-21_myg-13/README.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+authors: [Kouteki]
+---
+
+# The More You Gno 13: Test5, gnoverse and more
+
+This edition of "The More You Gno" is packed with updates and highlights. Test5 testnet was launched; a new community space called gnoverse is now available. The mainnet launch milestone is up and the development is speeding up. It's an exciting time to be a part of the gno.land project.
+
+## Test5 is live
+
+On November 13 we launched Test5, a new iteration of the multinode testnet. The validator set has been expanded from 7 to 17 nodes, plus a number of non-validator nodes. It also boasts GovDAO v2 that expands the on-chain voting capabilities, and a number of fixes and quality of live improvements for the developers.
```
Comment (leohhhn) [inline posts/2024-11-21_myg-13/README.md:14]:
```suggestion
On November 13 we launched Test5, a new iteration of the multinode testnet. The validator set has been expanded from 7 to 17 nodes, plus a number of non-validator nodes. It also boasts GovDAO v2 that expands the on-chain voting capabilities, and a number of fixes and quality of life improvements for the developers. 
```

File: posts/2024-11-21_myg-13/README.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+# The More You Gno 13: Test5, gnoverse and more
+
+This edition of "The More You Gno" is packed with updates and highlights. Test5 testnet was launched; a new community space called gnoverse is now available. The mainnet launch milestone is up and the development is speeding up. It's an exciting time to be a part of the gno.land project.
+
+## Test5 is live
+
+On November 13 we launched Test5, a new iteration of the multinode testnet. The validator set has been expanded from 7 to 17 nodes, plus a number of non-validator nodes. It also boasts GovDAO v2 that expands the on-chain voting capabilities, and a number of fixes and quality of live improvements for the developers. 
+
+Test5 is already available on Adena and Faucet Hub, so feel free to give it a try.
```
Comment (leohhhn) [inline posts/2024-11-21_myg-13/README.md:16]:
```suggestion
Test5 is already available on Adena and the Faucet Hub, so feel free to give it a try.
```

File: posts/2024-11-21_myg-13/README.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
+---
+publication_date: 2024-11-21T00:00:00Z
+slug: monthly-dev-13
+tags: [gnoland, ecosystem, updates, gnovm]
+authors: [Kouteki]
+---
+
+# The More You Gno 13: Test5, gnoverse and more
+
+This edition of "The More You Gno" is packed with updates and highlights. Test5 testnet was launched; a new community space called gnoverse is now available. The mainnet launch milestone is up and the development is speeding up. It's an exciting time to be a part of the gno.land project.
```
Comment (leohhhn) [inline posts/2024-11-21_myg-13/README.md:10]:
```suggestion
This edition of "The More You Gno" is packed with updates and highlights. Test5 testnet was launched; a new community space called `gnoverse` is now available. The mainnet launch milestone is up and the development is speeding up. It's an exciting time to be a part of the gno.land project.
```

File: posts/2024-11-21_myg-13/README.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+This GitHub space maintains a strong connection with the [gnolang](https://github.com/gnolang) community, fostering collaboration and resource sharing among developers and enthusiasts. Our goal is to support and highlight projects that enhance the gno.land experience, and to be much more hands-off in the process.
+
+We value community contributions! Whether you’re a developer, designer, or simply enthusiastic about Gno, your input matters. Join us by submitting pull requests, sharing ideas, or engaging in discussions. If you need a new repo for your project, we'd be happy to oblige.
+
+## Mainnet launch
+
+While we're getting an official announcement ready, take a sneak peek at the [gno.land mainnet launch milestone](https://github.com/gnolang/gno/milestone/7) on GitHub. The scope has been finalized for the most part, and our engineering teams are covering a lot of ground. We'll most likely launch at least one more testnet between now and the mainnet launch, so stay tuned.
```
Comment (leohhhn) [inline posts/2024-11-21_myg-13/README.md:28]: Nice

**Review by leohhhn — APPROVED** [review]
lgtm, just a few typos

**Pull request merged**